linalg/solvers.cpp

```diff
@@ -1319,6 +1319,54 @@ void NewtonSolver::Mult(const Vector &b, Vector &x) const
    final_norm = norm;
 }
 
+void NewtonSolver::CheckJacobian(Vector &x, Array<int> &ess_tdof)
+{
+   MFEM_ASSERT(oper != nullptr, "Call SetOperator first.");
+
+   double eps = 1e-8;
```

Comment:
I recommend making this an argument to the method, so users can decide what tolerance they want to accept. (Apologies for the random drive-by comment from someone who is not reviewing the PR.)